focus-ios/focus-ios-tests/XCUITest/DragAndDropTest.swift

```diff
@@ -5,20 +5,19 @@
 import XCTest
 
 class DragAndDropTest: BaseTestCase {
```

Comment:
I haven't found another file to put this test, so I let this test to be in this file instead.